infrastructure/interface/include/interface/msg_translation.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <ml4kp_bridge/defs.h>
+
+#include <prx_models/mj_mushr.hpp>
+
+namespace interface
+{
+
+inline void translate_msg(prx_models::MushrPlan& mushr_plan, const ml4kp_bridge::Plan& plan)
```

Comment:
Since this is specific to MuSHR, does it make sense to be here, or in `prx_models`? (Ok if this dependency order makes sense)

Reply:
Yeah, is because of the dependencies:  it here is for `interface` to depend on both `ml4kp_brdige` and `prx_models`. If we have it on `prx_models`, it would be:  `ml4kp_brdige` -> `prx_models` -> `interface`. I would prefer to have a _single layer_ of dependencies.